src/api/axiosInstance.ts

```
@@ -118,32 +106,40 @@ axiosInstance.interceptors.response.use(
     const originalRequest = config as AxiosRequestConfig & { _retry?: boolean };
 
     if (response?.status === 401 && originalRequest && !originalRequest._retry) {
+      // 리프레시 요청 자체에서의 401은 재시도/리프레시를 하지 않고 즉시 실패 처리
+      try {
+        const base = axiosInstance.defaults.baseURL ?? window.location.origin;
+        const reqUrl = originalRequest.url ?? '';
+        const pathname = reqUrl.startsWith('http') ? new URL(reqUrl).pathname : new URL(reqUrl, base).pathname;
+        if (pathname === ENDPOINTS.auth.refresh) {
+          return Promise.reject(error);
+        }
+      } catch {}
```

ESLint

[error] Empty block statement.

(no-empty)